deploy/namespace.yaml

```diff
@@ -4,3 +4,6 @@ metadata:
   name: $NAMESPACE
   labels:
     name: $NAMESPACE
+    pod-security.kubernetes.io/enforce: privileged
+    pod-security.kubernetes.io/enforce-version: v1.24
+    security.openshift.io/scc.podSecurityLabelSync: "false"
```

Comment:
this change needed for dev setups ? or there are other consumers of the deploy folder ?
why do we need to add openshift labels here  ?

Reply:
This is needed for "make deploy-setup". I don't believe there are other consumers. Unfortunately openshift 4.12 with the new pod security feature needs the openshift label.

Reply:
it would be great if we have these 3 labels added to namespace only when when `make deploy-setup` (i.e deployed in openshift) and not always since when running `make deploy-setup-k8s` these labels are not needed.

Reply:
Let's create a new namespace file for Openshift. And only apply this new namespace file in the case of openshift (deploy-setup and openshift namespace) in the Makefile.

Reply:
I think the fact that it is required or not depends on how k8s is being configured, not if it is Openshift or not.

Reply:
Hmm maybe it is better to use kubectl labels for the different scenarios.

Reply:
Let's just remove the last label and use something similar to https://github.com/openshift/dpu-network-operator/pull/52/commits/0e8e5511fb0183b1452a8e68af1d2c12e622ea08#diff-ce0d597a9e5f20a8fe61c3ba5e6185a6b90cfef2c00bb5b5ef3e97b15b79f480R6-R8